searx/engines/chinaso.py

```diff
@@ -7,6 +7,10 @@
 from searx.exceptions import SearxEngineAPIException
 from searx.utils import html_to_text
 
+from typing import TYPE_CHECKING
+if TYPE_CHECKING:
+    from typing import Literal
```

Comment:
Import `typing.TYPE_CHECKING` to import `typing.Literal` only when `TYPE_CHECKING` is true makes no sense .. just simply:

```python
import typing
```